packages/nx/src/project-graph/project-graph.ts

```diff
@@ -240,7 +240,7 @@ export class ProjectGraphError extends Error {
   }
 }
 
-function handleProjectGraphError(opts: { exitOnError: boolean }, e) {
+export function handleProjectGraphError(opts: { exitOnError: boolean }, e) {
```

Comment:
It doesn't look like you ended up using this.

Reply:
it's used in `graph.ts` to show errors where we can't recover a partial graph.